pkg/apis/security/v1alpha1/types.go

```diff
@@ -216,7 +216,7 @@ type Tier struct {
 type TierSpec struct {
 	// Priority specfies the order of the Tier relative to other Tiers.
 	// Currently supporting only 5 Tiers. i.e. 1-5 range for Priority.
-	Priority int32 `json:"priority"`
+	Priority uint8 `json:"priority"`
```

Comment:
Is better to keep it uint32 and if needed validate in the web hook?

Reply:
i think i already updated this in the latest PR.. kept it uint32

Reply:
Ok